UniConnectApp/app/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,5 @@
     <string name="app_name">UniConnect</string>
     <string name="title">title</string>
     <string name="description">description</string>
+    <string name="post_title">Create a Post</string>
```

Comment:
Added a new string within the strings.xml file. When creating any other constant strings, such as this one for the page's title, I would suggest placing them in this file rather than in the code itself.